exercises/tweet.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+import sys
+import redis
+import time
+import json
+
+HOST = "redis-13418.c8.us-east-1-4.ec2.cloud.redislabs.com"
+PORT = 13418
+PASS = "QCqZr7caZxWcUahORl0ReOg2JvqD8HZ4"
+POST_PREFIX = "posts:"
+FOLLOWEES_PREFIX = "followees:"
+USER = 'user'
+MESSAGE = 'message'
+TIME = 'time'
+
+
+def post(user: str, message: str, conn):
+    post_to_add = {MESSAGE: message, TIME: time.time()}
+
+    list_name = POST_PREFIX + user
+    message = json.dumps(post_to_add)
+    conn.rpush(list_name, message)
+    result = json.loads(conn.lindex(list_name, -1))
```

Comment:
It's a bit non-standard to use JSON for storing the data in Redis. This is also what we do in RCP, but that's not a good reason 😄 .
It would be better to use a native Redis data structure, namely a hash, to store the data.

Reply:
Ok, and we should keep only the key of the hash in the list right?